CHANGELOG.md

```diff
@@ -2,12 +2,109 @@
 
 ## Table of Contents
 
+- [v0.4.0-rc1](#v040-rc1)
 - [v0.3.0](#v030)
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+
+## v0.4.0-rc1
+
+API version: v1alpha2
+
+This release is intended to be last alpha release, so there are a lot of breaking API changes. Please read these release notes carefully.
+
+### Major Changes
+
+* The Gateway API APIGroup has moved from `networking.x-k8s.io` to `gateway.networking.k8s.io`. This means that, as far as the apiserver is concerned, this version is wholly distinct from v1alpha1, and automatic conversion is not possible. As part of this process, Gateway API is now subject to Kubernetes API review, the same as changes made to core API resources. More details in [#780](https://github.com/kubernetes-sigs/gateway-api/pull/780) and [#716](https://github.com/kubernetes-sigs/gateway-api/issues/716).
```

Comment:
Please wrap these lines to 80 char columns like the rest of the file.

Reply:
dang, I thought I had fixed all these. Will do.